includes/wp-cli/class-orchestrate-runner.php

```diff
@@ -87,7 +87,9 @@ public function run_event( $args, $assoc_args ) {
 			$error_data = $run->get_error_data();
 
 			if ( isset( $error_data['status'] ) && 404 === $error_data['status'] ) {
-				\WP_CLI::success( $run->get_error_message() );
+				\WP_CLI::warning( $run->get_error_message() );
+
+				exit;
```

Comment:
Should this be `return` rather than `exit`, just to be safe? I'm not sure if this function would be called anywhere else.

Reply:
The `exit` is fine. This function shouldn't be called anywhere outside of WP-CLI; there are helpers in the `Events` class for those situations.